CONTRIBUTING.md

```diff
@@ -244,15 +124,15 @@ If you'd like to contribute to the design or front-end, feel free to contribute
 
 Documentation is automatically synced from master to the [Gutenberg Documentation Website](https://wordpress.org/gutenberg/handbook/) every 15 minutes.
 
-To add a new documentation page, you'll have to create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder and add an item to the [root-manifest.json](https://github.com/WordPress/gutenberg/blob/master/docs/root-manifest.json).
+To add a new documentation page, you'll have to create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder and add an item to the [toc.json](/docs/toc.json).
```

Comment:
Is it `toc.json` now? It has changed and I tried to figure out based on the tool which generated the manifest file.

Reply:
@chrisvanpatten